R/package-auth.R

```diff
@@ -259,5 +282,17 @@ db_oauth_client <- function(host = db_host()) {
 
 }
 
-
-
+#' Returns the default config profile
+#' @details Returns the config profile first looking at `DATABRICKS_CONFIG_PROFILE`
+#' and then the `db_profile` option.
+#' 
+#' @return profile name
+#' @export
```

Comment:
I'm okay with this being un-exported unless you feel strongly that its helpful exported?